res/values-zh-rCN/aosip_arrays.xml

```diff
@@ -0,0 +1,129 @@
+    <string-array name="app_notification_sound_timeout_entries" translatable="false">
+        <item>@string/app_notification_sound_timeout_value_none</item>
+        <item>@string/app_notification_sound_timeout_value_10_seconds</item>
+        <item>@string/app_notification_sound_timeout_value_30_seconds</item>
+        <item>@string/app_notification_sound_timeout_value_1_minute</item>
+        <item>@string/app_notification_sound_timeout_value_5_minutes</item>
+        <item>@string/app_notification_sound_timeout_value_15_minutes</item>
+        <item>@string/app_notification_sound_timeout_value_30_minutes</item>
+    </string-array>
+
+    <!-- Do not translate. -->
+    <string-array name="app_notification_sound_timeout_values" translatable="false">
+        <item>0</item>
+        <item>10000</item>
+        <item>30000</item>
+        <item>60000</item>
+        <item>300000</item>
+        <item>900000</item>
+        <item>1800000</item>
+    </string-array>
+
+    <!-- Fingerprint sensor locations.  Devices should configure their sensor location using
+         config_fingerprintSensorLocation -->
+    <string-array name="security_settings_fingerprint_sensor_locations" translatable="false">
+        <item>@string/security_settings_fingerprint_enroll_find_sensor_message</item>
+        <item>@string/security_settings_fingerprint_enroll_find_sensor_message_front</item>
+        <item>@string/security_settings_fingerprint_enroll_find_sensor_message_left</item>
+        <item>@string/security_settings_fingerprint_enroll_find_sensor_message_right</item>
+    </string-array>
+
+    <string-array name="display_cutout_style_entries" translatable="false">
+        <item>@string/display_cutout_style_normal</item>
+        <item>@string/display_cutout_style_immerse</item>
+        <item>@string/display_cutout_style_hide</item>
+    </string-array>
+
+    <string-array name="display_cutout_style_values" translatable="false">
+        <item>0</item>
+        <item>1</item>
+        <item>2</item>
+    </string-array>
+
+    <!-- Refresh rate -->
+    <string-array name="refresh_rate_entries">
+        <item>@string/auto_refresh_rate</item>
+        <item>60Hz</item>
+        <item>120Hz</item>
+    </string-array>
+    <string-array name="refresh_rate_values">
+        <item>0</item>
+        <item>1</item>
+        <item>2</item>
+    </string-array>
+
+    <!-- Vibration patterns -->
+    <string-array name="vibration_pattern_entries">
+        <item>@string/pattern_dzzz_dzzz</item>
+        <item>@string/pattern_dzzz_da</item>
+        <item>@string/pattern_mm_mm_mm</item>
+        <item>@string/pattern_da_da_dzzz</item>
+        <item>@string/pattern_da_dzzz_da</item>
+        <item>@string/pattern_custom</item>
+    </string-array>
+    <string-array name="vibration_pattern_values">
+        <item>0</item>
+        <item>1</item>
+        <item>2</item>
+        <item>3</item>
+        <item>4</item>
+        <item>5</item>
+    </string-array>
```

Comment:
```suggestion
```
Do not include values you don't really translate. Especially do not include those marked as non translateable